components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/mgwdefinition/MgwAPIDefinition.java

```diff
@@ -117,4 +119,22 @@ public String getResponseInterceptor() {
     public void setResponseInterceptor(String responseInterceptor) {
         this.responseInterceptor = responseInterceptor;
     }
+
+    /**
+     * Identify if the API information used or not.
+     *
+     * @return true if the API information is used inside
+     */
+    public boolean getIsUsed(){
```

Comment:
Formatting issue

Reply:
the commented code is no longer available.